todo/__manifest__.py

```diff
@@ -37,7 +37,7 @@
         ####################################################
         ##      Javascript files to build React app       ##
         ####################################################
-        'todo.assets': [
+        'web.assets_frontend': [
```

Comment:
Thanks for contributing. The rest look good, but I'm not sure about this. It was meant to be a separate bundle hence a different bundle name